bin/code-extractor

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+require File.expand_path(File.join("..", "lib", "cfme-versions"), __FILE__)
```

Comment:
Don't think this should be here?

Reply:
Oh s---!

Good catch... oops!

(Can you tell were I was pulling some copy-pasta code from? 😆 )